pandas_vet/__init__.py

```diff
@@ -83,6 +83,13 @@ def run(self):
 
 
 def check_import_name(node: ast.Import) -> List:
+    """Check AST for imports of pandas not using the preferred alias 'pd'.
+
+    Error/warning message to recommend use of 'pd' alias.
+
+    :param node:   an AST node of type Import
```

Comment:
Looks like you still need to change these to a single space between the name and the description. Looks like they are tabs at the moment.

```suggestion
    :param node: an AST node of type Import
    :return errors: list of errors of type PD001 with line number and column offset
```